tests/acceptance/features/bootstrap/TrashbinContext.php

```diff
@@ -307,50 +307,36 @@ public function userGetsFilesInTheTrashbinWithDepthUsingTheWebdavApi(string $use
 	}
 
 	/**
-	 * @Then the trashbin DAV response should not contain these nodes
+	 * @Then /^the trashbin DAV response should (not|)\s?contain these nodes$/
```

Comment:
we should prefer readability also
```suggestion
	 * @Then /^the trashbin DAV response (should|should not) contain these nodes$/
```